src/main/java/com/browserstack/automate/ci/common/tracking/PluginsTracker.java

```diff
@@ -27,14 +27,42 @@ public PluginsTracker(final String username, final String accessKey) {
         this.username = username;
         this.accessKey = accessKey;
         this.trackingId = Tools.getUniqueString(true, true);
+        initializeClient();
     }
 
     public PluginsTracker() {
         this.username = null;
         this.accessKey = null;
         this.trackingId = Tools.getUniqueString(true, true);
+        initializeClient();
     }
-
+    
+    public void initializeClient(){
+
+        String username = JenkinsProxySettings.getUsername();
+        String password = JenkinsProxySettings.getPassword();
+        Authenticator proxyAuthenticator = new Authenticator() {
+            @Override public Request authenticate(Route route, Response response) throws IOException {
```

Comment:
We can probably follow a single convention everywhere while adding annotations for functions. i.e. `@Override` and function signature in separate lines etc.

Reply:
Done